packages/utils/src/Crypto.ts

```diff
@@ -365,18 +267,3 @@ export function makeKeypairFromUri<
   const keyring = new Keyring({ ss58Format, type: type ?? 'ed25519' })
   return keyring.addFromUri(uri) as KiltKeyringPair & { type: KeyType }
 }
-
-/**
- * Generate from a seed a x25519 keypair to be used as DID encryption key.
- *
- * @param seed The keypair seed, only optional in the tests.
- * @returns The keypair.
- */
-export function makeEncryptionKeypairFromSeed(
-  seed = randomAsU8a(32)
-): KiltEncryptionKeypair {
-  return {
-    ...naclBoxPairFromSecret(seed),
-    type: 'x25519',
-  }
-}
```

Comment:
I get removing the encrypt/decrypt stuff, but don't you think we should keep key creation around as we allow a certain set of public key types on DIDs and removing this would mean x25519 keys are the only ones that cannot be created using the sdk?

Reply:
This is more of a discussion PR. If these keys are only usable for messaging, and the messaging is deprecated, probably we don’t need to support them by the SDK directly. If you think they’d still be useful, let’s close this :)

Reply:
yeah I would rather keep it around for now. x25519-based key agreement is a use case for our DIDs, and even if there are other encryption schemes based on that besides the one implemented here, I think it's fair to have at least one supported by the sdk.